src/pages/get-started/configure-commerce.md

```diff
@@ -88,6 +88,9 @@ Commerce provides two sources for events: observers and plugins. You must specif
    ```bash
    bin/magento events:subscribe observer.customer_login --fields=firstname --fields=lastname --fields=entity_id
    ```
+   
+   **Warning**: If you subscribe to events with `events:subscribe` command the configuration will be stored in `app/etc/config.php` and you should keep in mind that this file can be replaced with the `app/etc/config.php` file from Git during deployment so as result the event subscription will be lost as well.
```

Comment:
@oshmyheliuk Does this warning apply to the event provider configuration for a Commerce app deployed on cloud as well?

Reply:
No, the event provider configuration is not stored in `app/etc/config.php`, it's stored in the database, so it won't be accidentally erased during redeployment.
This warning is only related to event subscriptions via the CLI command.